src/xrpld/rpc/detail/TransactionSign.cpp

```diff
@@ -222,6 +223,22 @@ checkPayment(
             rpcINVALID_PARAMS,
             "Cannot specify both 'tx_json.Paths' and 'build_path'");
 
+    std::optional<uint256> domain;
+    if (tx_json.isMember(jss::domain))
```

Comment:
we're mxing `jss::domain` and `sfDomainID`, it seems to be a bug and it's not unit tested

Reply:
This API is deprecated anyways. I'll change it to sfDomainID. Unit test is not really important here.

Reply:
I'd suggest using jss::domain instead of sfDomainID to maintain consistency